lib/recurly/pricing/calculations.js

```diff
@@ -176,6 +184,37 @@ Calculations.prototype.addons = function () {
 };
 
 /**
+ * Attaches usage addon info to the price object
+ *
+ * @private
+ */
+
+Calculations.prototype.usageAddons = function(){
+  var usageAddons = filter(this.items.plan.addons,function(addon){
+    return addon.add_on_type === "usage";
+  });
+
+  each(usageAddons, function (addon) {
```

Comment:
we can use `usageAddons.forEach` here. These shims were necessary when this module was first written, but out browser support doesn't necessitate it anymore.